tests/by_image/base-notebook/test_healthcheck.py

```diff
@@ -145,18 +141,6 @@ def test_not_healthy(
     env: list[str] | None,
     cmd: list[str] | None,
 ) -> None:
-    running_container = container.run_detached(
-        tty=True,
-        environment=env,
-        command=cmd,
-    )
-
-    # giving some time to let the server start
-    finish_time = time.time() + 5
```

Comment:
`5` is actually a bug here, because this way it tests nothing - if I put above, it's not enough to healthcheck to become green in tests